src/libsystemd/sd-bus/sd-bus.c

```diff
@@ -521,7 +521,7 @@ void bus_set_state(sd_bus *bus, enum bus_state state) {
         if (state == bus->state)
                 return;
 
-        log_debug("Bus %s: changing state %s → %s", strna(bus->description), table[bus->state], table[state]);
+        log_debug("Bus %s: changing state %s -> %s", strna(bus->description), table[bus->state], table[state]);
```

Comment:
Why not to use special_glyph() here?

Reply:
Hmm, why not indeed... `libsystemd` can use stuff from `basic`... Updated.